interface/patient_file/encounter/forms.php

```diff
@@ -20,6 +20,10 @@
     $groupId = $therapy_group;
 }
 $attendant_id = $attendant_type == 'pid' ? $pid : $therapy_group;
+if(!acl_check("groups","glog",false, array('view','write'))){
+    echo xlt("access not allowed");
```

Comment:
I don't understand this. It appears you are always denying access to the encounter summary if the user does not have access to groups/glog.  Is that what you want, and if so why?